chemprop/cli/hyperopt.py

```diff
@@ -1,16 +1,348 @@
+import json
+import logging
+import sys
 from argparse import ArgumentParser, Namespace
+from pathlib import Path
 
+import torch
+from lightning import pytorch as pl
+from lightning.pytorch.loggers import CSVLogger, TensorBoardLogger
+from lightning.pytorch.callbacks import ModelCheckpoint, EarlyStopping
+from ray import tune
+from ray.train import CheckpointConfig, RunConfig, ScalingConfig
+from ray.train.lightning import (RayDDPStrategy, RayLightningEnvironment,
+                                 RayTrainReportCallback, prepare_trainer)
+from ray.train.torch import TorchTrainer
+from ray.tune.schedulers import ASHAScheduler
+
+from chemprop.cli.common import (add_common_args, process_common_args,
+                                 validate_common_args)
+from chemprop.cli.train import (add_train_args, build_datasets, build_model,
+                                build_splits, normalize_inputs,
+                                process_train_args, validate_train_args)
 from chemprop.cli.utils.command import Subcommand
+from chemprop.data import MolGraphDataLoader
+from chemprop.nn import AggregationRegistry
+from chemprop.nn.utils import Activation
```

Comment:
Some of them are imported but not used.

Reply:
If we merge https://github.com/chemprop/chemprop/pull/714 before this PR we can all see which can be removed via GitHub actions

Reply:
I have removed unused imports